source/cloud/aws/eks.md

```diff
@@ -20,24 +20,32 @@ $ aws configure
 
 ## Create the Kubernetes cluster
 
-Now we can launch a GPU enabled EKS cluster. First launch an EKS cluster with `eksctl`.
+Now we can launch a GPU enabled EKS cluster with `eksctl`.
+
+```{note}
+1. You will need to create or import public ssh-key to be able to execute the following command.
+In your aws console under `EC2` in the side panel under Network & Security > Key Pairs, you can create a
+key pair or import (see "Actions" dropdown) one you've created locally.
+
+2. If you are not using your default AWS profile, add `--profile <your-profile>` to the following command.
+```
 
 ```console
 $ eksctl create cluster rapids \
-                      --version 1.29 \
+                      --version 1.30 \
                       --nodes 3 \
-                      --node-type=p3.8xlarge \
+                      --node-type=g4dn.xlarge \
                       --timeout=40m \
                       --ssh-access \
-                      --ssh-public-key <public key ID> \  # Be sure to set your public key ID here
+                      --ssh-public-key <public key ID> \  # Named assigned during creation of your key in aws console
                       --region us-east-1 \
                       --zones=us-east-1c,us-east-1b,us-east-1d \
                       --auto-kubeconfig
 ```
 
 With this command, you’ve launched an EKS cluster called `rapids`. You’ve specified that it should use nodes of type `p3.8xlarge`. We also specified that we don't want to install the NVIDIA drivers as we will do that with the NVIDIA operator.
 
-To access the cluster we need to pull down the credentials.
+To access the cluster we need to pull down the credentials. Add `--profile <your-profile>` if you are not using the default profile.
```

Comment:
```suggestion
To access the cluster we need to pull down the credentials.
Add `--profile <your-profile>` if you are not using the default profile.
```

Could you move this sentence to its own line? That formatting for most markdown renderers is the same (will still be a paragraph with wrapping), but breaking it up across multiple lines makes editing a little easier and the git blame a little more useful as a source of "when did this last line change?".